b2sdk/v2/bucket.py

```diff
@@ -21,6 +28,68 @@ def get_fresh_state(self) -> 'Bucket':
         except v3BucketIdNotFound as e:
             raise BucketIdNotFound(e.bucket_id)
 
+    @_file_infos_rename
+    def upload_bytes(
+        self,
+        data_bytes,
+        file_name,
+        content_type=None,
+        file_info: dict | None = None,
+        progress_listener=None,
+        encryption: v3.EncryptionSetting | None = None,
+        file_retention: v3.FileRetentionSetting | None = None,
+        legal_hold: v3.LegalHold | None = None,
+        large_file_sha1: Sha1HexDigest | None = None,
+        custom_upload_timestamp: int | None = None,
+        cache_control: str | None = None,
+    ):
+        return super().upload_bytes(
+            data_bytes=data_bytes,
+            file_name=file_name,
+            content_type=content_type,
+            file_info=file_info,
+            progress_listener=progress_listener,
+            encryption=encryption,
+            file_retention=file_retention,
+            legal_hold=legal_hold,
+            large_file_sha1=large_file_sha1,
+            custom_upload_timestamp=custom_upload_timestamp,
+            cache_control=cache_control,
+        )
```

Comment:
depends on IDE I guess

original author of the argument rename function seems to suggest this is the safe way to do it 
https://github.com/reef-technologies/b2-sdk-python/blob/3fd20f18423a057472d4e7be95df004bd994a249/b2sdk/v0/account_info.py#L29-L49

I will play with it some more and see

Reply:
I managed to get it working, but it breaks the test and I would have to rewrite them all without just mocking wrapped v3 methods 

```
def test_bucket__upload_file__supports_file_infos(dummy_bucket, file_info):
    """Test v2.Bucket.upload_file support of deprecated file_infos param"""
    with patch.object(v3.Bucket, 'upload_local_file') as mock_method,\
 ```
 since `upload_file = _file_infos_rename(v3.Bucket.upload_file)` is bound during module load, this patch happens just too late to mock v3.Bucket.upload_file


Not sure if this is worth it

Reply:
as concluded on slack - not worth the time to change it